pandas/tests/tools/test_merge.py

```diff
@@ -1224,7 +1220,7 @@ def f():
         expected = (
             DataFrame(dict(
                 household_id=[1, 2, 2, 2, 2, 3, 3, 3, 3, 3, 3, 4],
-                asset_id=["nl0000301109", "nl0000289783", "gb00b03mlx29",
```

Comment:
I think this is a typo in the original test. I am quite sure because the deleted ID (nl0000289783) does not appear in any dataframe of _'test_join_multi_levels2_' function.